pages/why-vacate.tsx

```diff
@@ -35,8 +35,9 @@ export default function WhyVacatePage() {
         imgsrc={content.heroBanner.imgsrc}
       />
 
-      <SectionContainer>
+      <SectionContainer id="main-section">
```

Comment:
Consider changing this to something more specific like `why-vacate-main` but besides that I think it's all good to go